engine/src/gfx/mesh.h

```diff
@@ -1,20 +1,20 @@
 /*
  * mesh.h
  *
- * Copyright (C) 2001-2002 Daniel Horn and Alan Shieh
- * Copyright (C) 2002-2019 Daniel Horn, klaussfreire, pheonixstorm, safemode, dan_w,
- *  ace123, jacks, and other Vega Strike contributors
```

Comment:
Explicitly named contributors should probably remain explicitly named

Reply:
Fixed all occurrences of explicitly named contributors in .h files (around 70 files in all)